src/modules/flow/http-client/http-client.json

```diff
@@ -47,7 +47,7 @@
           },
           {
             "data_type": "string",
-            "default": "application/json",
+            "default": "text/stream",
```

Comment:
I'd make this a list of both, with text/stream in the highest priority. That way if the server doesn't accept text/stream, like some HTTP, then we get the json or text.